ui/src/components/shared/Detail.tsx

```diff
@@ -18,7 +18,7 @@ export function Detail({
   text: string;
 }) {
   const value = ellipsisPosition ? text.slice(0, text.length - ellipsisPosition) : text;
-  const isUrl = z.string().url().safeParse(text).success;
+  const isUrl = text.startsWith("http://") || text.startsWith("https://");
```

Comment:
What about `ftp://`?

Reply:
I think we only need to cover `http` and `https`. The list of [URL schemes](https://en.wikipedia.org/wiki/List_of_URI_schemes) is huge but why should we care about any other?

Reply:
The moment that we need more protocols, we'll add it